zk/tests/test_zk.py

```diff
@@ -78,6 +78,23 @@ def test_error_state(aggregator, dd_environment, get_conn_failure_config):
     aggregator.assert_metric(mname, value=1, count=1, tags=["mytag"])
 
 
+def test_parse_replica_mntr(aggregator, mock_mntr_output, get_test_instance, caplog):
+    def mntr_side_effect(command):
+        if command == "mntr":
+            return mock_mntr_output
+
+    expected_message = "Unexpected 'mntr' output"
+    check = ZookeeperCheck(conftest.CHECK_NAME, {}, [get_test_instance])
+    check._send_command = mock.MagicMock(side_effect=mntr_side_effect)
+
+    caplog.clear()
+    caplog.set_level(logging.WARNING)
+
+    assert expected_message not in caplog.text
```

Comment:
I thought with the fixture we are expecting a log message? ( in the except line 380)

Reply:
Took a different approach than using caplog.text